sample/build.gradle

```diff
@@ -25,8 +26,12 @@ dependencies {
   compile "com.android.support:recyclerview-v7:$supportLibVersion"
   compile "com.android.support:design:$supportLibVersion"
 
+  compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+
   testCompile 'junit:junit:4.12'
 
   androidTestCompile project(':library')
-
+}
+repositories {
+  mavenCentral()
```

Comment:
I don't think that adding mavenCentral() is needed for the subprojects. We already add jcenter() to all subprojects from the main build.gradle file.